packages/auth/src/core/errors.ts

```diff
@@ -528,6 +528,7 @@ export const AUTH_ERROR_CODES_MAP_DO_NOT_USE_INTERNALLY = {
   INVALID_EMAIL: 'auth/invalid-email',
   INVALID_EMULATOR_SCHEME: 'auth/invalid-emulator-scheme',
   INVALID_IDP_RESPONSE: 'auth/invalid-credential',
+  INVALID_LOGIN_CREDENTIALS: 'auth/invalid-credential',
```

Comment:
Should this be "auth/invalid-login-credentials", the missing code reported in https://github.com/firebase/firebase-js-sdk/issues/7661?

Reply:
We are reusing the existing error code 'auth/invalid-credential' since it is very similar to the backend error and is already public on the SDK.

Reply:
Is there a reason not to create a new error type?

Reply:
> I am using firebase 10.6.0 and it is returning auth/invalid-login-credentials when there is an incorrect username or password. Will this change to start returning auth/invalid-credential in later updates?

That's correct. It is currently converting the server error "INVALID_LOGIN_CREDENTIALS" to lower-case and sending that error code - https://github.com/firebase/firebase-js-sdk/blob/bebecdaad7fa552505055ab7705da478203078b6/packages/auth/src/api/index.ts#L204

We will now populate the error map so the server error maps to "auth/invalid-credential"

> Is there a reason not to create a new error type?

This is to avoid creating another error code that is very similar to the existing invalid-credential code and to be consistent across iOS, Android and Web. On Android, we are reusing the https://firebase.google.com/docs/reference/android/com/google/firebase/auth/FirebaseAuthInvalidCredentialsException rather than create a new one, so following a similar approach on Web. Do you see any issue with this approach?

Changing behavior from auth/invalid-login-credentials  (a non-exposed error code) to auth/invalid-credential (a public error code) can be a breaking change, but i am not sure, since the previous error code was not public. WDYT @DellaBitta ?

Reply:
Sounds like a good reason to me. Thanks for the clarification!